.eslintrc

```diff
@@ -26,5 +28,13 @@
         "import/no-extraneous-dependencies": ["error", { "devDependencies": ["**/*.mdx"] }] // Allow importing docz React components in *.mdx files
       }
     }
-  ]
+  ],
+  "parser": "@babel/eslint-parser",
+  "rules": {
```

Comment:
I think we only need the line `"import/prefer-default-export": "off",` here.
All other can be removed and all tests still pass.